presentations/2025-06-02_replaymeeting.py

```
@@ -0,0 +1,59 @@
+            continue
+        genotype = get_genotype(mouse_name)
+        sessions = get_mouse_sessions(mouse_name)
+        # try:
+        #     sessions = get_mouse_sessions(mouse_name)
+        # except Exception as e:
+        #     print(f"Error retrieving sessions for {mouse_name}: {e}")
+        #     continue
+
+        genotype_dict[genotype].append(mouse_name)
+
+    with open("replay_meeting_data.json", "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)